apps/backoffice-v2/public/config.js

```diff
@@ -0,0 +1,11 @@
+globalThis.env = {
+  VITE_API_URL: "http://google.com",
+  VITE_API_KEY: "secret",
+  VITE_AUTH_ENABLED: true,
+  VITE_MOCK_SERVER: false,
+  VITE_POLLING_INTERVAL: 10,
+  VITE_ASSIGNMENT_POLLING_INTERVAL: 5,
+  VITE_FETCH_SIGNED_URL: false,
+  VITE_ENVIRONMENT_NAME: "local",
+  MODE: "production"
+};
```

Comment:
should use default value if preset, if case someone set his value while running the docker container so you should consider the provided value:

```
globalThis.env = {
  VITE_API_URL: import.meta.env.VITE_API_URL || 'http://google.com',
  VITE_KYB_DEFINITION_ID: import.meta.env.VITE_KYB_DEFINITION_ID || 'kyb_parent_kyc_session_example',
  VITE_API_KEY: import.meta.env.VITE_API_KEY || 'secret',
  VITE_ENVIRONMENT_NAME: import.meta.env.VITE_ENVIRONMENT_NAME || 'local',
  VITE_DEFAULT_EXAMPLE_TOKEN: import.meta.env.VITE_DEFAULT_EXAMPLE_TOKEN || '12345678-1234-1234-1234-123456789012',
  VITE_SENTRY_AUTH_TOKEN: import.meta.env.VITE_SENTRY_AUTH_TOKEN || '',
  VITE_SENTRY_DSN: import.meta.env.VITE_SENTRY_DSN || '',
};
```